### ISSUE TITLE
Update all the existing tickets using templates

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3991]
List all the tickets excluding the ones in the icebox column:

- [ ] ticket .. @christinach 
- [ ] ticket .. @sandbergja

**Comment by christinach** [comment]
Created a label 'template-update' and will label all the tickets that they're missing a template.
